src/configuration/GPG/GnuPG/params.txt

```diff
@@ -0,0 +1,9 @@
+Key-Type: RSA
+Key-Length: 8192  
```

Comment:
I think 4096 bit keys should be sufficient for RSA, even for the next 10-20 years. GPG keys should expire long before that date anyway.